### PR TITLE
Add support PHP 8.1

### PR DESCRIPTION
**Pull request by reindert-vetter**

### CODE REVIEW

File: lib/apt-install-php-ubuntu.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ then
          php${version}-zip
 fi
 
-if [[ $release = 'focal' && `echo "$version < 8.0" | bc` == 1 ]]
+if [[ $release = 'focal' && `echo "$version < 8.1" | bc` == 1 ]]
```
Comment (nanasess) [inline lib/apt-install-php-ubuntu.sh:44]:
```suggestion
if [[ $release = 'focal' && `echo "$version < 8.0" | bc` == 1 ]]
```
The JSON and xmlrpc extensions exist only under PHP 8.0. This fix is probably unnecessary.
https://github.com/nanasess/setup-php/runs/5008646783?check_suite_focus=true#step:7:512

**Review by nanasess — CHANGES_REQUESTED** [review]

**Comment by nanasess** [comment]
@reindert-vetter Welcome to setup-php, thanks for the Pull Request!
There is an error in GitHub Actions, please check it.

**Comment by nanasess** [comment]
PHP8.0.3 build failure issue fixed in #63

**Comment by reindert-vetter** [comment]
> PHP8.0.3 build failure issue fixed in #63

Thank you. I just looked at it, but I couldn't fix it.

**Review by nanasess — APPROVED** [review]

**Comment by nanasess** [comment]
LGTM

**Comment by nanasess** [comment]
@reindert-vetter 
Thanks you so much!, Fixed version has been released.
https://github.com/nanasess/setup-php/releases/tag/v3.0.9

**Comment by reindert-vetter** [comment]
I really like this package to work. This package deserve more stars.

**Pull request merged**